sccm/core/servers/deploy/configure/azure-services-wizard.md

```diff
@@ -209,6 +209,12 @@ Select **Sign in** to authenticate to Azure as an administrative user. These cre
 
 Select **OK** to create the native app in Azure AD and close the Create Client Application dialog. This action returns to the [Client App dialog](#client-app-dialog).
 
+### Renew secret key Azure AD apps
+Prior to 1806 version, to renew secret key of an Azure App, you need to recreate the App.
+On 1806 and above:
+Created App --> renewal will be done by entering azure credentials, and adding the info on the renew secret key wizard
```

Comment:
Let's do these as a bulleted list, and use a colon instead of the arrow. I don't have access to a console right now, but isn't there an option in the ribbon to renew key? We should explicitly mention that, in bold. For example:

```suggestion
- Created app: Go to the **Azure AD** node, and select the tenant. In the details pane, select the app, and then select **Renew Secret Key** in the ribbon. Entering your Azure credentials, and add the info on the renew secret key wizard.
```

Reply:
Sorry I should have been more clear. A bulleted list starts with a dash: `-`
For example:
```
- Created app: ...  
- Imported app: ...  
```

Reply:
Also, please capitalize "Azure" at the end of line 216:
"Enter your Azure credentials"